archiver-lib/src/archiver.rs

```diff
@@ -171,21 +171,23 @@ impl Archiver {
     /// Causes panic if none
     /// * `node` - The archiver node
     /// * `cluster_entrypoint` - ContactInfo representing an entry into the network
-    /// * `keypair` - Keypair for this archiver
+    /// * `archiver_keypair` - Keypair for this archiver
     #[allow(clippy::new_ret_no_self)]
     pub fn new(
         ledger_path: &Path,
         node: Node,
         cluster_entrypoint: ContactInfo,
-        keypair: Arc<Keypair>,
+        archiver_keypair: Arc<Keypair>,
```

Comment:
Cleanup like this is great, please just pull it out into a separate PR and it can land immediately